Engine/ScriptAnalyzer.cs

```diff
@@ -1522,16 +1522,18 @@ public IEnumerable<DiagnosticRecord> AnalyzeScriptDefinition(string scriptDefini
                 return null;
             }
 
-            if (errors != null && errors.Length > 0)
+            var relevantParseErrors = RemoveTypeNotFoundParseErrors(errors);
```

Comment:
Should there be an escape hatch here? Is there a scenario where a user would like to see all of the parse errors? I have a feeling that this rule is actually useful (as it catches type errors like `[System.Stirng]`).

Reply:
Maybe rather output a warning instead of a verbose message? Or even return a non-terminating error?

Reply:
I changed it to output a warning for the moment.

Reply:
The errors aren't terminating anyway, except that when you get too many of them PSSA gives up. When scanning some of the Azure packages for the gallery (https://dtlgalleryint.cloudapp.net/packages/AzSKStaging/3.1.2 is an example), we get about 4000 errors like this, so I'd really welcome some option to suppress them.

Reply:
@edyoung Do you get other errors apart from the `TypeNotFound` error as well? This PR removes all of them and only outputs them as a warning instead. Yes, when there are more than 10 non-TypeNotFound, then PSSA will throw a terminating error. Maybe an environment variable to tweak this magic value?

Reply:
I think it's fine to terminate parsing after 10 errors, but TypeNotFound issues not treated as errors shouldn't count towards that limit. Right now those are the only messages I get for that package but in general there will be other errors. 

Actually you should be able to download that package if you like: 
```
Register-PSRepository -Name GalleryRolling -SourceLocation https://dtlgalleryint.cloudapp.net/api/v2/ 
Save-Module -Repository GalleryRolling -Name AzSKStaging -AllowPreRelease -Path whatever
```

Reply:
I tried it with your branch, and the errors are converted to warnings as expected.

Reply:
Yes, TypeNotFound does not count towards that limit, this PR changes it to exactly that, only a warning (like `Write-Warning` not a rule violation!) is emitted to the console host. Thanks for the snippet.